include/qpid/dispatch/amqp.h

```diff
@@ -19,6 +19,8 @@
  * under the License.
```

Comment:
This Arm64 fail. I am not sure if that is new, but I don't remember seeing it before

https://app.travis-ci.com/github/apache/qpid-dispatch/jobs/559394005#L4289

```
25: ======================================================================
25: ERROR: test_forbid_fallback (system_tests_policy.PolicyTerminusCapabilities)
25: ----------------------------------------------------------------------
25: Traceback (most recent call last):
25:   File "/home/travis/build/apache/qpid-dispatch/tests/system_tests_policy.py", line 496, in test_forbid_fallback
25:     br1 = BlockingConnection(self.router.addresses[0])
25:   File "/home/travis/build/apache/qpid-dispatch/qpid-proton/python/proton/_utils.py", line 394, in __init__
25:     self.wait(lambda: not (self.conn.state & Endpoint.REMOTE_UNINIT),
25:   File "/home/travis/build/apache/qpid-dispatch/qpid-proton/python/proton/_utils.py", line 533, in wait
25:     raise ConnectionException(
25: proton._exceptions.ConnectionException: Connection amqp://0.0.0.0:21926 disconnected: Condition('amqp:connection:framing-error', 'Expected SASL protocol header: no protocol header found (connection aborted)')
25: 
25: ----------------------------------------------------------------------
25: Ran 27 tests in 106.160s
25: 
25: FAILED (errors=1)
25/76 Test #25: system_tests_policy ...............................***Failed  106.29 sec
```

Reply:
May or may not be related to the patch - hard to tell w/o more context.  I wonder if this was caused by a router crash?

Reply:
Router seems to be exiting cleanly, the log says

```
25: Router QDR.Policy output file:
25: >>>>
25: -----------------------------------------------------
25: Suppressions used:
25:   count      bytes template
25:       1         56 ^IoAdapter_init$
25:       2         46 ^load_server_config$
25:       6        192 ^qd_policy_c_counts_alloc$
25:     591     901140 /libpython3.*.so
25: -----------------------------------------------------
```

That to me suggest the exit code of the router was also checked.

I think it could be a case of the stalls/freezes as in https://issues.apache.org/jira/browse/DISPATCH-2094, except there a timeout in the tests allows to fail a single test and carry on. So, quite possibly this is not new problem introduced by PR.